packages/backendai-ui/README.md

```diff
@@ -0,0 +1,151 @@
+# backend.ai-ui
+
+This is a react component project for WebUI.
+
+## How to setup relay
+
+> [!NOTE]
+> This project contains components related to Relay. Before using these components, please ensure that your project is properly set up with a suitable Relay environment, if necessary, by following the steps below.
+
+1. Set up Relay as a multi-project and configure it so that this project can be compiled.
+
+   ```js
+   // relay.config.js or relay.config.json
+   module.exports = {
+     root: '.',
+     sources: {
+       'packages/backendai-ui': 'backendai-ui',
+       'your-project-path': 'your-project',
+     },
+     excludes: ['**/node_modules/**', '**/__mocks__/**', '**/__generated__/**'],
+     projects: {
+       'backendai-ui': {
+         language: 'typescript',
+         schema: 'schema-path',
+         output: 'packages/backendai-ui/src/__generated__',
+         eagerEsModules: true,
+         ...options,
+       },
+       'your-project': {
+         language: 'your language',
+         schema: 'schema-path',
+         output: 'your-project-path/output-path',
+         base: 'backendai-ui', // to use backendai-ui's fragment
+         ...options,
+       },
+     },
+   };
+   ```
+
+2. Run the relay-compiler and make sure that the `backendai-ui` project compiles successfully.
+
+   ```console
+   $ relay-compiler
+   ```
+
+3. Depending on the bundler environment, you may need to set up an `alias` for `__generated__`. With the following configuration, relay-compiler will be able to correctly resolve the `__generated__` path.
+
+   ```ts
+   // vite.config.ts
+   export default defineConfig({
+    resolve: {
+      alias: {
+        // This is used to resolve the __generated__ directory for Relay
+        // Since relay uses the directory './__generated__' internally, change this to your-project-path/__generated__.
```

Comment:
`map this to` is better


```suggestion
        // Since relay uses the directory './__generated__' internally, map this to your-project-path/__generated__.
```